RESOURCES/INTHEWILD.md

```diff
@@ -181,3 +181,4 @@ Join our growing community!
 - [Twitter](https://twitter.com/)
 - [VLMedia](https://www.vlmedia.com.tr/) [@ibotheperfect]
 - [Yahoo!](https://yahoo.com/)
+- [Onebeat](https://1beat.com/) [@GuyAttia]
```

Comment:
Please place the entry in the correct slot based on alphabetical ordering

Reply:
Done